libparsers/tests/csv_rw_route/preprocessor.cpp

```diff
@@ -0,0 +1,154 @@
+#include "csv_rw_route.hpp"
+#include "utils.hpp"
+#include <boost/filesystem/fstream.hpp>
+#include <doctest.h>
+
+using namespace std::string_literals;
+namespace bf = boost::filesystem;
+namespace cs = parsers::csv_rw_route;
+namespace p_util = parsers::util;
+namespace {
```

Comment:
There should be a space above the beginning of a namespace, and there shouldn't be one after it.